### DISCUSSION TITLE
Cannot publish streaming on Chrome

### DISCUSSION THREAD

**Discussion by tuongtran** [ant-media/Ant-Media-Server#6627] [Q&A]
Hi,

I cannot publish stream via webRTC on Chrome, below are screenshots on [ant media test](https://test.antmedia.io:5443/).

<img width="1331" alt="1" src="https://github.com/user-attachments/assets/a39ab95f-2f37-41e0-bd42-a36dda33a5eb">
<img width="1497" alt="2" src="https://github.com/user-attachments/assets/ad67c87b-6dee-4b75-8b30-86d330f1d204">
<img width="1087" alt="3" src="https://github.com/user-attachments/assets/9f62d247-4ad3-4bdb-ab6b-d9bec7a6c0ce">

I have tried
- Use / install the latest ant media version (Community Edition 2.11.1 20240831_1437)
- Test with old versions (v2.6 CE & EE)
- Configure TURN server on both Antmedia control panel & client side (Javascript)
But results are same. Got publishTimeout error.

Test env.
- Mac pro M2
- Mac OS 13.4 (22F66)
- Chrome latest version Version 128.0.6613.114 (Official Build) (arm64)

Firefox & Safari work fine. This issue has happened since last week.

Any ideas or configurations, please help?

**Comment by rahul7827** [comment]
Hello @tuongtran 

I tried this on my windows system, and is working fine. Have you tried switching your network ? 
Also if you have made any changes "**chrome://flags/**" here. Reset them all and try again.

**Comment by tuongtran** [comment]
I have reset chrome://flags/, not working

when troubleshooting I see

Got `iceConnectionState` failed error

Disable video of mediaConstraints -> it works

```
 mediaConstraints: {
        video: false,
        audio: true
      },

```

Retest on FF & Safari -> doesn't work.

**Comment by tuongtran** [comment]
Tested on Mac 14.6.1 (23G93)

Found issues on Chrome

I use `Touch VPN` chrome extension (v5.0.18). After disabled, I can publish stream

Tested on [ant media test page](https://test.antmedia.io:5443/) -> worked

Chrome
Ant media 2.6 EE -> worked
Ant media 2.6 CE -> worked
Ant media CE 2.11.1 20240831_1437 -> not working. However trickle-ice is working now. I will do more testing on this version later

Safari
Ant media 2.6 EE -> worked
Ant media 2.6 CE -> worked
Ant media CE 2.11.1 20240831_1437 -> not working

FF
Ant media 2.6 EE -> worked
Ant media 2.6 CE -> worked
Ant media CE 2.11.1 20240831_1437 -> not working

I will recheck firewall if blocks UDP/TCP ports on the latest version and update if any.

**Comment by tuongtran** [comment]
After double check, it was incorrect turn server url when I setup directly on the ant media panel.
I use coturn, so the configuration should look like `turn:turn-domain.com:3478?transport=tcp`

Everything is working fine now.

**Reply by amarantmeida** [reply]
@tuongtran Glad to know that you got it to work. Keep testing and do let us know if you need any further assistance.
